_includes/tutorials/anomaly-detection/ksql/markup/dev/create-accounts-to-monitor-table.adoc

```diff
@@ -0,0 +1,21 @@
+For this use case, let's say that a single transaction to one of the companies in the suspicious_names table is probably okay, but multiple transactions to one or more of those companies in a 24-hour period is an anomaly. ksqlDB gives us the ability to see if any anomalies are present for a particular user with the following query.
+
+[source,sql]
+----
+CREATE TABLE accounts_to_monitor
+    WITH (kafka_topic='accounts_to_monitor', partitions=1, value_format='JSON') AS
+    SELECT TIMESTAMPTOSTRING(WINDOWSTART, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_START, // <1>
+           TIMESTAMPTOSTRING(WINDOWEND, 'yyyy-MM-dd HH:mm:ss Z') AS WINDOW_END,
+           USERNAME
+    FROM suspicious_transactions
+    WINDOW TUMBLING (SIZE 24 HOURS) // <2>
+    GROUP BY USERNAME
+    HAVING COUNT(*) > 3; // <3>
+----
++++++
+<div class="colist arabic">
+<p><i class="conum" data-value="1"></i><b>1</b> The timestamps here are important, they tell us what interval of time suspicious activity occurred.</p>
```

Comment:
`they tell us what interval of time suspicious activity occurred` -- the timestamps themselves don't tell the interval.  Would it be more accurate to phrase, something along the lines of, `they are used to calculate the interval of time between transactions` (or however you see fit)

Reply:
hm rereading this, I think I should have said "The fields `WINDOW_START` and `WINDOW_END`" rather than "timestamps". The number maps to the field creation of the windows. @ybyzek what are your thoughts on:
>The fields `WINDOW_START` and `WINDOW_END` tell us what interval of time suspicious activity occurred.

Reply:
Actually, I misread as well! (I mistakenly was thinking of the timestamps of the transaction itself). Yes, your proposed text disambiguates.

Reply:
Change applied.